integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.service.spec.ts

```diff
@@ -147,6 +148,7 @@ describe('OpfCheckoutPaymentWrapperService', () => {
       Promise.resolve()
     );
     spyOn(service, 'renderPaymentGateway').and.callThrough();
+    spyOn<any>(service, 'storePaymentSessionId');
```

Comment:
cast to 'any' because storePaymentSessionId is a protected method.
same reason for below 'service a any' (line 199).
Alternative is to define storePaymentSessionId as public but it would impact customer.